test/fixtures/lightscript/arrow-methods/class-malformed-async-skinny-arrow/actual.js

```diff
@@ -0,0 +1,3 @@
+class X {
+  async method() -> await a
```

Comment:
Womp, sorry, should have seen this before – we should remove (or at least rename) this test since it no longer tests what it says it does